aether-odk-module/aether/odk/api/xform_utils.py

```diff
@@ -323,22 +323,22 @@ def parse_xform_to_avro_schema(xml_definition, default_version=DEFAULT_XFORM_VER
                             {
                                 'name': 'latitude',
                                 'doc': _('latitude'),
-                                'type': __get_avro_primitive_type('float', True),
+                                'type': __get_avro_primitive_type('float', required=False),
```

Comment:
Could you add a comment saying something like "Since an Avro schema does not contain the same branching logic as an XForm, a field that is mandatory in a form is not actually always present, and therefore cannot be required in the schema"?

Reply:
Done!

Reply:
But a couple of lines below :grimacing: